### PR TITLE
fix for holders

### PR DESCRIPTION
**Pull request by vonHacht**
ref: https://github.com/ranaroussi/yfinance/issues/178

### CODE REVIEW

File: yfinance/utils.py [hunk 1]
```diff
@@ -26,7 +26,6 @@
 import pandas as _pd
 import numpy as _np
 import sys as _sys
-import re as _re
```
Comment (vonHacht) [inline yfinance/utils.py]: double import of regex

File: yfinance/base.py [hunk 2]
```diff
@@ -168,7 +168,7 @@ def history(self, period="1mo", interval="1d",
                 print('- %s: %s' % (self.ticker, err_msg))
             return shared._DFS[self.ticker]
 
-        elif "chart" not in data or data["chart"]["result"] is None or \
```
Comment (vonHacht) [inline yfinance/base.py]: Codefactor: _Unnecessary_ "elif" after "return"_

File: yfinance/base.py [hunk 3]
```diff
@@ -283,13 +283,14 @@ def cleanup(data):
...
         holders = _pd.read_html(url)
         self._major_holders = holders[0]
-        self._institutional_holders = holders[1]
-        if 'Date Reported' in self._institutional_holders:
-            self._institutional_holders['Date Reported'] = _pd.to_datetime(
-                self._institutional_holders['Date Reported'])
-        if '% Out' in self._institutional_holders:
-            self._institutional_holders['% Out'] = self._institutional_holders[
-                '% Out'].str.replace('%', '').astype(float)/100
+        if len(holders) > 1:
```
Comment (vonHacht) [inline yfinance/base.py]: Solution: check length of holders

Comment (mariomarine) [reply yfinance/base.py]: This would be useful to me.

**Comment by AlexanderKonstantinidis** [comment]
Is it possible to provide the problematic .py files corrected--referring to them by their original names-- so I replace them?  Moreover, where I will find them in my disk?

**Comment by vonHacht** [comment]
See: #178

**Comment by stockjanitor** [comment]
Hello sir, I am having the empty dataframe issue, I saw this can solve the issue, but i dont understand anything in this thread, could you explain a little bit? i am a completely new in python

**Pull request closed without merging**